src/riak_kv_qry_compiler.erl

```diff
@@ -207,52 +246,44 @@ compile_select_col(DDL, {{window_agg_fn, FnName}, [FnArg1]}) ->
         Initial_state ->
             {ColTypes2, IsValid2, Compiled_arg1} =
                 compile_select_col_stateless(DDL, FnArg1),
-            %% all the windows agg fns so far are arity of 1
-            %% which we have forced in this clause by matching on a single argument in the
-            %% function head
             FnSig = riak_ql_window_agg_fns:get_arity_and_type_sig(FnName),
             {IsValid, ColRet} = check_types(FnName, FnSig, ColTypes2),
 
             IsValid3 = merge_validation(IsValid2, IsValid),
+            % all the windows agg fns so far are arity of 1
```

Comment:
On behalf of all cat^W emacs people, please keep this comment block marked with double `%`. Autoindenting this block in Emacs will move it far to the right. (Ditto for other blocks elsewhere.)